sys/kern/isa.c

```diff
@@ -0,0 +1,3 @@
+#include <sys/devclass.h>
+
+DEVCLASS_CREATE(isa);
```

Comment:
I think you could just move it to the end of `82371AB.c` (and remove this file).

Reply:
I could, but I'm not fully convinced that a generalized `isa` devclass should be created insde a specific driver.

Reply:
Okay, I understand. Let it be that way.